packages/core/src/utils/styleUtils/ssrStyles.ts

```diff
@@ -596,13 +692,36 @@ export const indexByBreakpoint = ({
       // TODO: Test this for nested patterns as the name might be just a random hash without the actual name (needs to be validated).
       variableName.startsWith('cfBackgroundImageUrl_')
     ) {
+      const width = resolveImageOptimizationOptions({
+        variableData: variables['cfWidth'],
+        defaultBreakpoint,
+        componentSettings,
+        componentVariablesOverwrites,
+      }) as string;
+
+      const options = resolveImageOptimizationOptions({
+        variableData: variables['cfBackgroundImageOptions'],
+        defaultBreakpoint,
+        componentSettings,
+        componentVariablesOverwrites,
+      }) as BackgroundImageOptions;
+
+      if (!options) {
+        console.error(
+          `Error transforming image asset: Required variable [cfBackgroundImageOptions] missing from component definition`,
+        );
+        continue;
```

Comment:
this looks suspicious tbh, don't we want to return a plain url if there is no options to optimize the image based on?

Reply:
Sorry, not quite following. It looks like we used to resolve a plain url in ssr, but now if there is no options we return nothing. Why not keeping a plain url for this case?

Reply:
So you saying that this shouldn't be the case, and the options should always be provided? that's fine with me 👍 thanks for providing the context

Reply:
Yes, e.g Currently on cavas currently if there is not `BackgroundImageOptions`, the image will not be rendered in the canvas, but then on SSR side ( on preview ) if we return a plain image url they can get confused and could think it as a bug on Canvas.